cypress/integration/page.spec.js

```diff
@@ -1026,7 +1104,7 @@
       page.spyOnCommentsPost();
 
       newCommentForm.emailInput().type('some@email.com');
-      newCommentForm.bodyArea().type('Some comment body')
+      newCommentForm.bodyArea().type('Some comment body');
```

Comment:
There is a missing semicolon at the end of the `cy.type` statement. Ensure that each statement is properly terminated to avoid syntax errors.